src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftCalls.kt

```diff
@@ -1,182 +1,6 @@
-            }
-            PartiQLValueType.MISSING -> {
-                log.error("PartiQL `MISSING` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.SYMBOL -> {
-                log.error("PartiQL `SYMBOL` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.INTERVAL -> {
-                log.error("PartiQL `INTERVAL` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.BLOB -> {
-                log.error("PartiQL `BLOB` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.CLOB -> {
-                log.error("PartiQL `CLOB` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.BAG -> {
-                log.error("PartiQL `BAG` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.LIST -> {
-                log.error("PartiQL `LIST` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.SEXP -> {
-                log.error("PartiQL `SEXP` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            PartiQLValueType.STRUCT -> {
-                log.error("PartiQL `STRUCT` type not supported in Redshift")
-                super.rewriteCast(type, args)
-            }
-            // using the customer type to rename type
-            PartiQLValueType.FLOAT32 -> exprCast(args[0].expr, typeFloat32())
-            PartiQLValueType.FLOAT64 -> exprCast(args[0].expr, typeFloat64())
-            PartiQLValueType.BINARY -> exprCast(args[0].expr, typeCustom("VARBYTE"))
-            PartiQLValueType.BYTE -> TODO("Mapping to VARBYTE(1), do this after supporting parameterized type")
-            else -> super.rewriteCast(type, args)
-        }
-    }
-
-    /**
-     * As far as the documentation goes, there is no indication that redshift support type assertion other than null.
-     * I.e., var IS NULL is supported, var IS INT2 may not.
-     * Also, there is seemingly no helper function like pg_typeof() in redshift either.
-     * Throwing a warning message if the type assertion is not targeting null type.
-     */
-    override fun isType(type: PartiQLValueType, args: SqlArgs): Expr {
-        when (type) {
-            PartiQLValueType.NULL -> Unit
-            else -> log.warn("Redshift does not support type assertion on ${type.name} ")
-        }
-        return super.isType(type, args)
-    }
-
-    private fun id(symbol: String) = identifierSymbol(symbol, Identifier.CaseSensitivity.INSENSITIVE)
-}
+public class RedshiftCalls(context: ScribeContext) : SqlCalls(context)
```

Comment:
self-review: for the other SQL dialects (Redshift, Trino, and Spark), only include the skeleton for the classes we'll implement in the future